tron/config/config_parse.py

```diff
@@ -372,6 +375,12 @@ class ValidateAction(Validator):
             valid_dict,
         'extra_volumes':
             build_list_of_type_validator(valid_volume, allow_empty=True),
+        'triggers_downstream':
+            valid_bool,
+        'triggered_by':
+            valid_dict,
```

Comment:
shouldn't this be a list?

Reply:
no, because we need to refer to triggers by both name and value to avoid situations where different triggers generate the same value

Reply:
What are the name and value in this case? I thought triggered_by would be a list of items like `MASTER.job.action.shortdate.%(shortdate-1)s`.

Reply:
my bad, i mixed up the attributes. `triggered_by` should be a list, `triggers_downstream` should be bool/dict. thanks 👍